packages/core/rendering/core-rendering-server-internal/src/rendering_service.tsx

```diff
@@ -151,6 +154,7 @@ export class RenderingService {
         faviconSVG: branding?.faviconSVG,
         faviconPNG: branding?.faviconPNG,
         pageTitle: branding?.pageTitle,
+        logo: branding?.logo,
```

Comment:
This is added because logo can also be changed on template.tsx right?

Reply:
If by changed you mean "needs to be read", then yes.